docs/quickstart-gke.md

```diff
@@ -13,12 +13,30 @@ This document walks you through setting up a Google Kubernetes Engine (GKE) clus
 * [kubecfg](https://github.com/ksonnet/kubecfg/releases)
 * [jq](https://stedolan.github.io/jq/)
 
+### DNS and G Suite requirements
+
+In addition to the requirements listed above, a domain name is also required for setting up Ingress endpoints to services running in the cluster. The specified domain name can be a top-level domain (TLD) or a subdomain. In either case you have to manually [set up the glue records](#configure-domain-registration-records)  to delegate DNS resolution queries to a Google Cloud DNS zone created and managed by BKPR.
+
+BKPR on GKE uses AuthZ domain based authorization, specified in `GCLOUD_AUTHZ_DOMAIN`, for verifying users as they login to the Elasticsearch, Kibana and Grafana dashboards. As such you need a [G Suite](https://gsuite.google.co.in/intl/en_in/) account configured set up for your authorization domain to enable users to login with their G Suite user accounts.
+
```

Comment:
@falfaro @anguslees Do you think we should note here that `GCLOUD_AUTHZ_DOMAIN` can optionally be configured to `*` to allow users to login with their @gmail.com email address?

Reply:
Just realized it's mentioned when `GCLOUD_AUTHZ_DOMAIN` is described later.

Reply:
We have a duty of care here and I think it is irresponsible for us to suggest that a user allow _all_ @gmail.com users access to their site - particularly when it won't be obvious that they've just done this.

I know we don't like requiring a Gsuite domain, but we need to fix that by exposing the oauth2_proxy email-address whitelist rather than suggesting (anywhere!) that they effectively just turn off authz.

If we _really_ want (and I don't think this is a good idea), we could implement an `--insecure` mode that doesn't use TLS, doesn't require real hostnames (no letsencrypt), and skips all of oauth2_proxy.  We still need hostnames in _some_ form if we continue to choose the service based on HTTP Host headers, but we could even drop that (to use multiple addresses or multiple ports) if we really want to drop all environmental requirements.

Reply:
I agree! I have updated the PR to remove the suggestion for specifying `GCLOUD_AUTHZ_DOMAIN="*"` to (kinda) bypass the g suite requirement